azure-pipelines.yml

```diff
@@ -120,7 +120,7 @@ jobs:
         jdkDirectory: '/Library/Java/JavaVirtualMachines/jdk-11.jdk/Contents/Home/'
         publishJUnitResults: false
         testResultsFiles: '**/TEST-*.xml'
-        tasks: 'shadowJar test'
+        tasks: 'check'
```

Comment:
You probably just want to straight up run the `build` task which will just build everything and run the `check` task for you.

Reply:
I think we're starting to stray too far from what this PR is for. Ci improvements can be made in separate PRs; the only reason the azure setup was modified here was because checkstyle got fixed in this PR